src/plugins/base/SUBTREELoader.js

```diff
@@ -873,4 +833,35 @@ export class SUBTREELoader extends LoaderBase {
 
 	}
 
+	/**
+	 * Generates the full external buffer URI for a tile by combining an implicit URI with a buffer URI.
+	 *
+	 * First, it parses the implicit URI using the tile properties and the provided template. Then, it creates a new URL
+	 * relative to the tile's base path, removes the last path segment, and appends the buffer URI.
+	 *
+	 * @param {Object} tile - The tile object that contains properties:
+	 *   - __level: the subdivision level,
+	 *   - __x, __y, __z: the tile coordinates,
+	 *   - __basePath: the base path for the external resources.
+	 * @param {string} uri - The URI template string with placeholders for the tile (e.g., `{level}`, `{x}`, `{y}`, `{z}`).
+	 * @param {string} bufUri - The buffer file name to append (e.g., "0_1.bin").
+	 * @returns {string} The full external buffer URI.
+	 */
+	parseImplicitURIBuffer( tile, uri, bufUri ) {
+
+		// Generate the base URI by replacing placeholders
+		const subUri = this.parseImplicitURI( tile, uri );
+
+		// Create a URL object relative to the tile's base path
+		const url = new URL( subUri, tile.__basePath + '/' );
+
+		// Remove the last path segment
+		url.pathname = url.pathname.substring( 0, url.pathname.lastIndexOf( '/' ) );
+
+		// Construct the final URL with the buffer URI appended
+		return new URL( url.pathname + '/' + bufUri, tile.__basePath + '/' ).toString();
+
+
+	}
```

Comment:
I don't love that this function has to access tile.__x etc and particularly tile.__basePath (since it's defined outside this file). I'll take at this a bit more this weekend to see if I can figure out another way to generate this url.